dockstore-integration-testing/src/test/java/io/dockstore/common/CommonTestUtilities.java

```diff
@@ -175,16 +192,47 @@ private static String getDockstoreToken(TestingPostgres testingPostgres, String
         return "Bearer " + (testingPostgres
                 .runSelectStatement("select content from token where tokensource='dockstore' and username= '" + username + "';", String.class));
     }
+    /**
+     * Deletes BitBucket Tokens from Database
+     *
+     * @param testingPostgres reference to the testing instance of Postgres
+     * @throws Exception
+     */
+    public static void deleteBitBucketToken(TestingPostgres testingPostgres)  {
```

Comment:
nitpick: think this can be private, at a glance look like this is only used in `CommonTestUtilities.java`